be/src/storage/rowset_column_update_state.h

```diff
@@ -0,0 +1,159 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+#include <string>
+#include <unordered_map>
+
+#include "storage/olap_common.h"
+#include "storage/primary_index.h"
+#include "storage/rowset/column_iterator.h"
+#include "storage/tablet_updates.h"
+
+namespace starrocks {
+
+class Tablet;
+class DeltaColumnGroup;
+class FileSystem;
+class Segment;
+class RandomAccessFile;
+class ColumnIterator;
+
+struct RowsetSegmentId {
```

Comment:
You can comment that what is rowset_id, rowset_seg_id and segment_id
It isn't very clear for me

Reply:
updated and comment, make it easy to understand.